From 213048fd9a5e800deb74cb5b7f0eaf465945c640 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabiano=20Fid=C3=AAncio?= <fidencio@redhat.com>
Date: Mon, 24 Apr 2017 21:09:02 +0200
Subject: [PATCH 122/135] RESPONDER_COMMON: Improve domaiN_resolution_order
 debug messages
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Indicate whether a domain_resolution_order has been used and where
it came from.

Signed-off-by: Fabiano Fidêncio <fidencio@redhat.com>

Reviewed-by: Pavel Březina <pbrezina@redhat.com>
---
 src/responder/common/responder_common.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/responder/common/responder_common.c b/src/responder/common/responder_common.c
index 62b71b5104fdbb585d086d44d2ca2ab9717dd788..7496d293fddb3e947d59a4f2aaeb2c83234dfcc7 100644
--- a/src/responder/common/responder_common.c
+++ b/src/responder/common/responder_common.c
@@ -1594,6 +1594,8 @@ errno_t sss_resp_populate_cr_domains(struct resp_ctx *rctx)
                 rctx, rctx->domains,
                 rctx->domain_resolution_order, &cr_domains);
         if (ret == EOK) {
+            DEBUG(SSSDBG_TRACE_FUNC,
+                  "Using domain_resolution_order from sssd.conf\n");
             goto done;
         } else {
             DEBUG(SSSDBG_MINOR_FAILURE,
@@ -1624,6 +1626,8 @@ errno_t sss_resp_populate_cr_domains(struct resp_ctx *rctx)
                                                        dom->sysdb,
                                                        &cr_domains);
         if (ret == EOK) {
+            DEBUG(SSSDBG_TRACE_FUNC,
+                  "Using domain_resolution_order from IPA ID View\n");
             goto done;
         }
 
@@ -1641,6 +1645,8 @@ errno_t sss_resp_populate_cr_domains(struct resp_ctx *rctx)
                                                   dom->sysdb, dom->name,
                                                   &cr_domains);
     if (ret == EOK) {
+        DEBUG(SSSDBG_TRACE_FUNC,
+              "Using domain_resolution_order from IPA Config\n");
         goto done;
     }
 
-- 
2.12.2

